src/pages/AccountRequest/RequestForm.tsx

```diff
@@ -0,0 +1,146 @@
+import FormInput from "components/Form/FormInput";
+import FormSelect from "components/Form/FormSelect";
+import { Form, Formik, FormikHelpers } from "formik";
+import React, { useEffect } from "react";
+import { IInstitution, IRole, IUserFormValues } from "./accountUtil";
+import * as Yup from "yup";
+import useAPI from "hooks/useAPI";
+import { Button, Col, InputGroup } from "react-bootstrap";
+import axios from "axios";
+
+const initialValues: IUserFormValues = {
+  role_id: 0,
+  username: "",
+  full_name: "",
+  email: "",
+  institution_id: 0,
+  introduction: "",
+};
+
+const validationSchema = Yup.object({
+  role_id: Yup.number().required("Required"),
+  username: Yup.string()
+    .required("Required")
+    .lowercase("Username must be lowercase")
+    .min(3, "Username must be at least 3 characters")
+    .max(20, "Username must be at most 20 characters"),
+  full_name: Yup.string().required("Required"),
+  email: Yup.string().required("Required").email("Invalid email format"),
+  institution_id: Yup.number().required("Required"),
+  introduction: Yup.string().required("Required"),
+});
+
+const RequestForm: React.FC = () => {
+  const { data: roles, sendRequest: fetchRoles } = useAPI();
+  const { data: institutions, sendRequest: fetchInstitutions } = useAPI();
+
+  useEffect(() => {
+    fetchRoles({
+      url: "/roles",
+      transformResponse: (response) => {
+        return JSON.parse(response);
+      },
+    });
+    fetchInstitutions({
+      url: "/institutions",
+      transformResponse: (response) => {
+        return JSON.parse(response);
+      },
+    });
+  }, [fetchRoles, fetchInstitutions]);
+
+  const onSubmit = async (values: IUserFormValues, submitProps: FormikHelpers<IUserFormValues>) => {
+    try {
+      const response = await axios.post("http://127.0.0.1:3000/api/v1/account_requests", values);
```

Comment:
It's more efficient to utilize the `useAPI` hook instead of generating a new Axios instance.